specification/contosowidgetmanager/data-plane/Azure.Contoso.WidgetManager/preview/2022-11-01-preview/widgets.json

```diff
@@ -285,22 +309,24 @@
                 "result": {
                   "$ref": "#/definitions/Widget",
                   "description": "The result of the operation."
-                },
-                "widgetName": {
```

Comment:
@weshaggard: This is flagged as a breaking change.  Intentional?

Reply:
This was caused by the change from cadl to typespec to generate this. I'm not sure if this is a real breaking change or not. That is a better question for the typespec team. @allenjzhang @markcowl any thoughts on why this was removed in the switch from cadl to typespec autorest emitter?

Reply:
It was a bug that it was there in the first place - the status monitor should not have the keys of the underlying resource as properties.